espnet2/enh/separator/asteroid_models.py

```diff
@@ -0,0 +1,144 @@
+from collections import OrderedDict
+from typing import Tuple
+
+import torch
+
+from espnet2.enh.separator.abs_separator import AbsSeparator
+
+
+class AsteroidModel_Converter(AbsSeparator):
+    def __init__(
+        self,
+        encoder_output_dim: int,
+        model_name: str,
+        num_spk: int,
+        pretrained_path: str = "",
+        loss_type: str = "si_snr",
+        **model_related_kwargs,
+    ):
+        """The class to convert the models from asteroid to AbsSeprator.
+
+        Args:
+            encoder_output_dim: input feature dimension, deafult=1 after the NullEncoder
+            num_spk: number of speakers
+            loss_type: loss type of enhancement
+            model_name: asteroind model names, e.g. ConvTasNet, DPTNet. Refers to
+                        https://github.com/asteroid-team/asteroid/
+                        blob/master/asteroid/models/__init__.py
+            pretrained_path: the name of pretrained model from Asteroid in Zenodo.
+                             Refers to: https://github.com/asteroid-team/asteroid/
+                             blob/master/docs/source/readmes/pretrained_models.md
```

Comment:
Rather point to the HF Hub: https://huggingface.co/models?filter=asteroid
We are transitioning, and some models are not there yet, but models should be added there from now on.